azure-bundle/LICENSE

```diff
@@ -203,107 +203,95 @@
 
 --------------------------------------------------------------------------------
 
-This binary artifact contains code from the following projects:
+This binary artifact contains:
```

Comment:
Seems a bit unnecesary to change this but I'm OK with it

Reply:
Let me revert this part to avoid the noise 😄

Reply:
Done !